nautobot/extras/tests/integration/test_plugins.py

```diff
@@ -122,3 +122,33 @@ def test_plugin_webhook_with_body(self):
         with open(os.path.join(tempfile.gettempdir(), "test_plugin_webhook_with_body"), "r") as f:
             self.assertEqual(json.loads(f.read()), {"message": "created"})
         os.remove(os.path.join(tempfile.gettempdir(), "test_plugin_webhook_with_body"))
+
+
+@skipIf(
+    "example_plugin" not in settings.PLUGINS,
+    "example_plugin not in settings.PLUGINS",
+)
```

Comment:
Shouldn't be needed - example_plugin should always be available during tests.

Reply:
I was following the example from above (the webhook test), I will remove this  line.

Reply:
Yup, we have a number of these that we need to clean up at some point. Just want to not make the existing tech debt worse. :-) Thanks.